build-tools/config/webpack/webpack.partial.conf.module.js

```diff
@@ -36,9 +34,6 @@ module.exports = ({ config, isDevelopment }) => webpackConfig => {
                 {
                   loader: 'sass-loader',
                   options: {
-                    importer: jsonImporter(),
```

Comment:
Does the json importing still works if this is removed?

Reply:
Sorry, I haven't tested that. The compiler was complaining that this option is not valid, so I removed it.

Reply:
They moved the node-sass options from the root to a special option called `sassOptions`. See https://github.com/webpack-contrib/sass-loader#sassoptions

Reply:
Moved it to the `sassOptions`